app/services/get_site_data.rb

```diff
@@ -0,0 +1,45 @@
+class GetSiteData
+  class<<self
+
+    def logger
+      @logger ||= Logger.new("log/testing.log")
+    end
+
+    def start_scraping_html(urls_data)
+      logger.info "HTML_SCRAPING_IS_STARTED_FROM_GET_SITE_DATA"
```

Comment:
no need to mention the function that triggered, each class and function represent a standalone component and thus you can't couple it together in logs either